client/src/main/scala/drt/client/components/TerminalDashboardComponent.scala

```diff
@@ -126,19 +126,20 @@ object TerminalDashboardComponent {
                 case _ => Icon.arrowRight
               }
 
-              <.div(^.className := s"queue-box col ${q.toString.toLowerCase} ${TerminalDesksAndQueuesRow.slaRagStatus(qWait, p.airportConfig.slaByQueue(q))}",
-                <.div(^.className := "queue-name", s"${Queues.displayName(q)}"),
-                <.div(^.className := "queue-box-text", Icon.users, s"$qPax pax joining"),
-                <.div(^.className := "queue-box-text", Icon.clockO, s"${MinuteAsAdjective(qWait).display} wait"),
-                <.div(^.className := "queue-box-text", waitIcon, s"queue time"),
+              <.dl(^.aria.label := s"Passenger joining queue ${Queues.displayName(q)}",
+                ^.className := s"queue-box col ${q.toString.toLowerCase} ${TerminalDesksAndQueuesRow.slaRagStatus(qWait, p.airportConfig.slaByQueue(q))}",
+                <.dt(^.className := "queue-name", s"${Queues.displayName(q)}"),
+                  <.dd(^.className := "queue-box-text", Icon.users, s"$qPax pax joining"),
+                  <.dd(^.className := "queue-box-text", Icon.clockO, s"${MinuteAsAdjective(qWait).display} wait"),
+                  <.dd(^.className := "queue-box-text", waitIcon, s"queue time")
               )
             }).toTagMod
           ),
 
-          <.div(^.className := "tb-bar row",
-            p.router.link(p.terminalPageTabLoc.copy(queryParams = Map("start" -> s"$urlPrevTime")))(^.className := "dashboard-time-switcher prev-bar col", Icon.angleDoubleLeft),
-            <.div(^.className := "time-label col", s"${start.prettyTime()} - ${end.prettyTime()}"),
-            p.router.link(p.terminalPageTabLoc.copy(queryParams = Map("start" -> s"$urlNextTime")))(^.className := "dashboard-time-switcher next-bar col", Icon.angleDoubleRight)
+          <.div(^.className := "tb-bar row", ^.aria.label := "Change time by selected time slot hours",
+            p.router.link(p.terminalPageTabLoc.copy(queryParams = Map("start" -> s"$urlPrevTime")))(^.aria.label := "Change time by previous selected time slot hours", ^.className := "dashboard-time-switcher prev-bar col", Icon.angleDoubleLeft),
```

Comment:
Could we re-word this so that it says what the time slot, ie `View previous x minutes` & `View next x minutes`?